compiler/luci/tests/test.lst

```diff
@@ -174,6 +174,8 @@ addread(Net_ZeroDim_001)
 # from res/CircleRecipes
 addread(CircleBatchMatMul_000)
 addread(InstanceNorm_000)
+addread(BCQFullyConnected_000)
+addread(BCQFullyConnected_001)
```

Comment:
Is BCQGather recipe not needed o going to be added?

Reply:
Because I had no reference until now, I didn't add it but will be added soon..!